src/copilot_aisdk/chat.py

```diff
@@ -24,11 +24,16 @@ async def get_documents(query, num_docs=5):
         credential=AzureKeyCredential(os.environ["AZURE_AI_SEARCH_KEY"]),
         index_name=os.environ["AZURE_AI_SEARCH_INDEX_NAME"])
 
+    aclient = AsyncAzureOpenAI(
```

Comment:
From Leah - why are we reconstructing this here instead of passing it from chat_completion function?